sdk/storage/azure-storage-blob/azure/storage/blob/aio/_blob_service_client_async.py

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
-
+# pylint: disable=invalid-overridden-method
```

Comment:
This does not solve the purpose. We must create a common base class for sync and async and inherit both the clients from the base class.


```
class BlobServiceClientBase():

class BlobServiceClient(BlobServiceClientBase):

again in ./aio
class BlobServiceClient(BlobServiceClientBase):
```

Reply:
The mypy pr is already doing this, I think we can reuse that one
Also this problem happened for upload, download, policies, I don't think we need to create a base and inherit  for sync and async for all cases.